habitat_baselines/config/default.py

```diff
@@ -193,5 +200,14 @@ def get_config(
         config.CMD_TRAILING_OPTS = config.CMD_TRAILING_OPTS + opts
         config.merge_from_list(config.CMD_TRAILING_OPTS)
 
+    if config.NUM_PROCESSES != -1:
```

Comment:
Use YACS.register_deprecated_keys() instead to handle this for us.

Reply:
This won't work because we are allowing new keys and YACS checks that before checking deprecation.